src/tests/posix/pthread/pthread_kill_test.c

```diff
@@ -15,36 +15,49 @@
 
 EMBOX_TEST_SUITE("posix/pthread_kill");
 
-void sighand(int signo) {
+static void sighand_a(int signo) {
     test_assert_not_zero(signo);
+    test_emit('a');
     return;
 }
 
-void *thread_test(void *parm) {
+static void sighand_b(int signo) {
+    test_assert_not_zero(signo);
+    test_emit('b');
+    return;
+}
+
+static void *thread_test(void *parm) {
     int rc = sleep(3);
     test_assert_not_zero(rc);
     return NULL;
 }
 
-void *thread_ref(void *parm) {
+static void *thread_ref(void *parm) {
     int rc = sleep(3);
     test_assert_zero(rc);
     return NULL;
 }
 
-void *thread_immediate_return(void *parm) {
+/*
+static void *thread_immediate_return(void *parm) {
```

Comment:
move it please where it's used

Reply:
I wanted to use it to generate a thread that terminates immediately. Since it returns immediately, we can guarantee the thread is terminated. Then we can test sending signal to a terminated thread.

Do you think I should delete it or leave it for later use?

Reply:
I'm not sure. But we try to add only used code.
As for this situation, maybe better way is to create a new issue and described the problem in it. Delete unused code to clean the PR

Reply:
How about I delete unused code, and mention the problem in another issue when I finish investigating the problem about sending signal to terminated threads?